api/src/main/java/org/openmrs/module/pihcore/setup/AuthenticationSetup.java

```diff
@@ -58,6 +58,7 @@ public static void setup(Config config) {
         whitelist.add("*.ico");
         whitelist.add("*.ttf");
         whitelist.add("*.woff");
+        whitelist.add("*.json");
```

Comment:
Well, if we have an endpoints that accept ".json" and return protected data, this would bypass authentication checks outside of any service layer.

Reply:
fair, we probably want to think about this a bit more...